src/zh/baozimanhua/src/eu/kanade/tachiyomi/extension/zh/baozimanhua/Baozi.kt

```diff
@@ -290,6 +290,7 @@ class Baozi : ParsedHttpSource(), ConfigurableSource {
             "cn.czmanga.com",
             "tw.czmanga.com",
             "www.czmanga.com",
+            "www.twmanga.com",
```

Comment:
```suggestion
            "cn.twmanga.com",
            "tw.twmanga.com",
            "www.twmanga.com",
```

`czmanga.com` now redirects to the new domain.

Reply:
Ok, I remove it